tensorflow/python/keras/metrics_test.py

```diff
@@ -1380,6 +1380,22 @@ def test_zero_and_non_zero_entries(self):
     expected_result = (0 + 1 / (1 + 1 - 1)) / 1
     self.assertAllClose(self.evaluate(result), expected_result, atol=1e-3)
 
+  def test_y_pred_as_logits_or_probability_tensor(self):
+    y_true = [2, 0, 1, 1]
+    y_pred = [[2.5, 0.2, 2.7], [-1.4, 0.7, 1.2], [2., 3., 1.4], [2.2, 2.4, 1.5]]
+    
+    m_obj = metrics.MeanIoU(num_classes=3)
+    self.evaluate(variables.variables_initializer(m_obj.variables))
+    result = m_obj(y_true, y_pred)
+
+    # cm = [[0., 0., 1.],
```

Comment:
Please include this section in the docstring as well, so that math of the example is more understandable.

Reply:
done, thanks